.github/workflows/build.yml

```diff
@@ -77,9 +77,9 @@ jobs:
           for f in secureboot.{{pk,null.pk,kek,db}.{crt,der,auth},db.arn,aws-efivars}; do
             ln -sr "cert/gardenlinux-$f" "cert/$f"
           done
-      - uses: actions/cache/save@1bd1e32a3bdc45362d1e726936510720a7c30a57 # pin@v3
+      - uses: actions/cache/save@1bd1e32a3bdc45362d1e726936510720a7c30a57 # pin@v4.2.0
         with:
-          path: cert
+          path: cert/*.*
```

Comment:
thanks for taking care of this.

Interesting that this action references actions/toolkit but do not seem to expose the `follow-symbolic-links` feature, at least I could not find it in actions/cache repo. 

Do you have a test run that you can link by any chance? 

Otherwise looks good to me

Reply:
I would assume that cert/*.* would force the action to follow the symlinks via wildcard (implemented by mentioned https://github.com/actions/toolkit/tree/main/packages/glob)